python/cuml/dask/linear_model/base.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common import raise_exception_from_futures
+from cuml.dask.common.comms import CommsContext
+from cuml.dask.common.input_utils import DistributedDataHandler
+from dask.distributed import wait
+
+from uuid import uuid1
+
+
+class BaseLinearModelSyncFitMixin(object):
+
+    def _fit(self, model_func, data, **kwargs):
```

Comment:
Nice to get all of this into the base class!

Reply:
Yep! We have a decomposition fit mixin coming in as well. After that, we're going to try to combine the fit mixin paths as much as possible.